### ISSUE TITLE
Queries not launching and stuck with some thread pre-set

### ISSUE THREAD

**Issue by betamigo98** [Garulf/Reddit-Browser#6]
Hi and many thanks for this service,
I do experience some error, that is activating the plugin only displays couple of pre-set subreddits.
I can't launch the search with the corresponding keyword I entered.

I checked the py file but no idea what to look for
I checked cache related item, thinking the pre-set come from such source

**Comment by Garulf** [comment]
Activating the plugin defaults to the "Popular" subreddit.

typing a `/` will pull up any favorited subreddit (You can save a subreddit by opening the context menu and selecting "Add subreddit to favorites")
![image](https://user-images.githubusercontent.com/535299/167451880-700d65f0-24a9-4b73-a80c-a3bf543e8fdd.png)


typing `/<reddit subreddit>` will pull up the most popular threads in that subreddit.

for example `r /flowlauncher`:
![image](https://user-images.githubusercontent.com/535299/167451280-d47cb420-18bd-46ad-835a-2b938e84758f.png)

**Comment by betamigo98** [comment]
Many for that answer that solve my issue.
I would suggest to to edit the documentation.
Also to note: I tried with "r/" without space in between in r and /, which doesn't get result (as it not supposed to do so, but it might add doubt when using for flow launcher for the first time).

**Comment by tdnzr** [comment]
Agreed, I recommend adding this info to the readme. It's currently not at all clear how to use this plugin.

**Comment by pratik-chakravorty** [comment]
This needs proper Readme support
